kuma/wiki/views/utils.py

```diff
@@ -36,22 +33,17 @@ def split_slug(slug):
 
     parent = '/'.join(slug_split)
 
-    return {
-        'specific': specific,
-        'parent': parent,
-        'full': slug,
-        'parent_split': slug_split,
-        'length': length,
-        'root': root,
-        'seo_root': seo_root,
+    return {                         # with this given: "some/kind/of/Path"
+        'specific': specific,        # 'Path'
+        'parent': parent,            # 'some/kind/of'
+        'full': slug,                # 'some/kind/of/Path'
+        'parent_split': slug_split,  # ['some', 'kind', 'of']
+        'length': length,            # 4
+        'root': root,                # 'some'
+        'seo_root': seo_root,        # 'some'
     }
 
 
-def join_slug(parent_split, slug):
-    parent_split.append(slug)
-    return '/'.join(parent_split)
-
-
 @newrelic.agent.function_trace()
```

Comment:
@darkwing It was amazing work at the time given constraints and time given to you. I'm impressed and will call you JavaScript *and* Python developer from now on.